errors/MissingTypeDeclaration.md

```diff
@@ -3,9 +3,11 @@
 ## Example
 
 ```purescript
-module ShortFailingExample where
+module Main where
+import Prelude
+import Effect.Console (log)
 
-...
+main = log "Main type is undefined."
```

Comment:
"undefined" isn't quite the right word to use here. The type is defined; it's inferrable, it can only ever be `Effect Unit`. It's just not written explicitly in the source file. How about "No type annotation has been provided." ?

Reply:
Sounds good to me!